scripts/ci/automation_full_test.py

```diff
@@ -29,7 +29,7 @@
 fix_failure_tests = sys.argv[5].lower() == 'true' if len(sys.argv) >= 6 else False
 target = sys.argv[6].lower() if len(sys.argv) >= 7 else 'cli'
 working_directory = os.getenv('BUILD_SOURCESDIRECTORY') if target == 'cli' else f"{os.getenv('BUILD_SOURCESDIRECTORY')}/azure-cli-extensions"
-azdev_test_result_dir = f"~/.azdev/env_config/mnt/vss/_work/1/s/env"
+azdev_test_result_dir = os.path.expanduser("~/.azdev/env_config/mnt/vss/_work/1/s/env")
```

Comment:
`open('~/.azdev/env_config/...', 'a')` with fail with:
```
No such file or directory: '~/.azdev/env_config/mnt/vss/_work/1/s/env/test_results_error_modules_1.txt'
```

Thus calling `os.path.expanduser` to parse `~` before file operation